schema.sql

```diff
@@ -1,5 +1,8 @@
-/* Database schema to keep the structure of entire database. */
-
 CREATE TABLE animals (
-    name varchar(100)
+    id int primary key not null,
```

Comment:
[OPTIONAL] Consider making this id auto incremented for ease  of inserting data into the table

Reply:
Thanks so much @DuaneDave 
I will make the necessary changes